tdisplays3-displaypages.yaml

```diff
@@ -0,0 +1,296 @@
+esphome:
+  name: "tdisplays3-displaypages"
+  friendly_name: "tdisplays3-displaypages"
+
+  on_boot: 
+    - priority: 600
+      then:
+        - lambda: |-
+            //id(disp).tft.init();
+            // Set the rotation here, not on the component
+            id(disp).tft.setRotation(3);
```

Comment:
Why must this be set here? It would be much nicer, if it would be able to use the component rotation, like other displays in esphome can do.
Is there a reason why the `setRotation` call can't be done in the `setup` method of the component? You might want to call a "set_rotation" method from the python file, to be able to set the rotation. You can find an example on the touchscreen code in this same repository.

Reply:
I'll put some more work into this.  You're right, I should figure out how to fix it correctly.

Reply:
Just replying to say I do have this figured out in another branch of my repository.  It's a bit strange because the rotations between ESPHome and TFT_eSPI don't align - they are always 90 degrees apart.  So I had to fetch the current config setting for the display device (in setup) and then change it to zero rotation and set the real rotation using the TFT_eSPI library based on what the original rotation was set to.  My plan (which lando might not even really want in his repo) is to be able to set an option for exposing the TFT objects, with default to false.  But you were right, there was a better way to do this.  I'm working on a branch that lets you define `expose_tft_objects: true`  as part of the display config and the rotation code would only run if that was true.